package.json

```diff
@@ -4,7 +4,7 @@
     "@wireapp/antiscroll-2": "1.3.1",
     "@wireapp/avs": "8.1.17",
     "@wireapp/core": "28.2.0",
-    "@wireapp/react-ui-kit": "8.7.0",
+    "@wireapp/react-ui-kit": "file:.yalc/@wireapp/react-ui-kit",
```

Comment:
I left it here since I'd need to do a bump of the ui-kit once the changes are published to npm as a last step. Ideally I probably should'nt commit my package.json and yarn.lock before the bump